i18n/fr/docusaurus-plugin-content-docs/version-23.10/graph-views/install-ha.md

```diff
@@ -0,0 +1,794 @@
+
+<Tabs groupId="sync">
+<TabItem value="Debian11" label="Debian 11">
+```bash
+apt install centreon-ha-common pcs pacemaker corosync corosync-qdevice
+```
+</TabItem>
+</Tabs>
+
+### Echange de clés SSH
+
+L'authentification basée sur une clé SSH doit être configurée pour que les fichiers et les commandes puissent être envoyés d'un nœud à l'autre par des comptes UNIX :
+
+* `mysql`
+
+Il existe deux façons d'échanger ces clés:
+
+* En utilisant la commande `ssh-copy-id`. Il doit pouvoir se connecter à l'hôte distant en utilisant un mot de passe. Il n'est cependant pas sûr que de tels comptes système disposent d'une authentification par mot de passe. Si vous choisissez cette méthode, nous vous conseillons de révoquer ce mot de passe par la suite avec la commande suivante : `passwd -d mysql`.
+* En copiant manuellement la clé publique dans `~/.ssh/authorized_keys`. Cette méthode est plus sûre.
+
+La deuxième méthode sera documentée ci-dessous.
+
+Pour le compte `mysql`, ces commandes doivent être exécutées sur les deux nœuds également :
+
+```bash
+systemctl stop mysql
+mkdir /home/mysql
+chown mysql: /home/mysql
+usermod -d /home/mysql mysql
+usermod -s /bin/bash mysql
+systemctl start mysql
+su - mysql
+```
+
+Pour le compte `mysql`, ces commandes doivent être exécutées sur les deux nœuds également :
+
+```bash
+ssh-keygen -t ed25519 -a 100
+cat ~/.ssh/id_ed25519.pub
+```
+
+Une fois cela fait, copiez le contenu du fichier de clé publique affiché par `cat` et collez-le dans `~/.ssh/authorized_keys` (doit être créé) sur l'autre nœud et appliquez les permissions correctes au fichier (toujours en tant qu'utilisateur `mysql`) :
+
+```bash
+chmod 600 ~/.ssh/authorized_keys
+```
+
+L'échange de clés doit être validé par une connexion initiale de chaque nœud à l'autre afin d'accepter et d'enregistrer l'empreinte SSH du nœud pair (toujours comme utilisateur `mysql`) :
+
+```bash
+ssh <peer node hostname>
+```
+
+Ensuite, quittez la session `mysql` en tapant `exit` ou `Ctrl-D`.
+
+## Configuration de la réplication des bases de données MariaDB
+
+Un cluster MariaDB primaire-secondaire sera mis en place afin que tout soit synchronisé en temps réel. 
+
+**Info**: sauf indication contraire, chacune des étapes suivantes doit être exécutée **sur les deux nœuds MAP**.
+
+### Configuration de MariaDB
+
+<Tabs groupId="sync">
+<TabItem value="Debian11" label="Debian 11">
+Pour des raisons d'optimisation et de fiabilité du cluster, vous devez ajouter ces options de réglage à la configuration de MariaDB dans le fichier `/etc/mysql/mariadb.conf.d/50-server.cnf`. Collez ces lignes (certaines doivent être modifiées) dans cette section :
+
+```ini
+[server]
+server-id=1 # SET TO 1 FOR PRIMARY AND 2 FOR SECONDARY
+#read_only
+log-bin=mysql-bin
+binlog-do-db=centreon_map
+innodb_flush_log_at_trx_commit=1
+sync_binlog=1
+binlog_format=MIXED
+slave_compressed_protocol=1
+datadir=/var/lib/mysql
+pid-file=/var/lib/mysql/mysql.pid
+
+ignore-db-dir=lost+found
```

Comment:
Should improve because this option will change in future (needed for ext4 FS on Debian) : 

```suggestion
ignore_db_dirs=lost+found
```